pkg/apis/options/alpha_options.go

```diff
@@ -42,7 +47,9 @@ type AlphaOptions struct {
 	MetricsServer Server `json:"metricsServer,omitempty"`
 
 	// Providers is used to configure multiple providers.
-	Providers Providers `json:"providers,omitempty"`
+	Providers provideropts.Providers `json:"providers,omitempty"`
+
+	TenantLoader tenant.LoaderConfiguration `flag:"tenant-loader" cfg:"tenant_loader"`
```

Comment:
This struct is meant to represent the configuration, please don't add runtime bits to it

Reply:
there _are_ configurations, not runtime structures,

Reply:
resolved. removed the runtime bits from options.